src/auto_archiver/modules/opentimestamps_enricher/__manifest__.py

```diff
@@ -30,34 +25,76 @@
             "help": "Optional whitelist of calendar servers. Override this if you are using your own calendar servers. e.g. ['https://mycalendar.com']",
             "type": "list"
         },
-        "verify_timestamps": {
-            "default": True,
-            "help": "Whether to verify timestamps after creating them.",
-            "type": "bool"
-        }
     },
     "description": """
     Creates OpenTimestamps proofs for archived files, providing blockchain-backed evidence of file existence at a specific time.
 
     Uses OpenTimestamps – a service that timestamps data using the Bitcoin blockchain, providing a decentralized 
-    and secure way to prove that data existed at a certain point in time.
+    and secure way to prove that data existed at a certain point in time. A SHA256 hash of the file to be timestamped is used as the token
+    and sent to each of the 'timestamp calendars' for inclusion in the blockchain. The proof is then saved alongside the original file in a file with
+    the '.ots' extension.
 
     ### Features
-    - Creates cryptographic timestamp proofs that link files to the Bitcoin blockchain
-    - Verifies existing timestamp proofs to confirm the time a file existed
-    - Uses multiple calendar servers to ensure reliability and redundancy
+    - Creates cryptographic timestamp proofs that link files to the Bitcoin or Litecoin blockchain
```

Comment:
I'd suggest not mentioning or supporting Litecoin attestations. While the OpenTimestamps protocol technically supports them, AFAIK no-one runs calendars that timestamp via Litecoin. Nor am I aware of any code available to even do that.

It's less confusing to just leave that off entirely given it's almost certain no-one is going to take advantage of it.